packages/lesswrong/components/tagging/TagSubforumPage2.tsx

```diff
@@ -148,13 +154,49 @@ export const styles = (theme: ThemeType): JssStyles => ({
   sidebarBoxWrapper: {
     backgroundColor: theme.palette.panelBackground.default,
     border: theme.palette.border.commentBorder,
-    paddingLeft: "1.5em",
-    paddingRight: "1.5em",
-    paddingBottom: "1em",
     marginBottom: 24,
   },
+  sidebarBoxWrapperDefaultPadding: {
+    padding: "1em 1.5em",
+  },
   tableOfContentsWrapper: {
     padding: 24,
+  },
+  membersListLink: {
+    background: 'none',
+    fontFamily: theme.typography.fontFamily,
+    fontSize: 13,
+    color: theme.palette.primary.main,
+    padding: 0,
+    '&:hover': {
+      opacity: 0.5
+    },
+    [theme.breakpoints.up('md')]: {
+      display: 'none' // only show on mobile (when the sidebar is not showing)
+    }
+  },
+  joinBtn: {
+    alignItems: 'center !important',
+    padding: '4px 0 0 0 !important',
+    '& button': {
+      minHeight: 0,
+      fontSize: 14,
+      padding: 8
+    },
+    [theme.breakpoints.down('sm')]: {
+      padding: '2px 0 0 0 !important',
```

Comment:
Why all the `!important`s? :(

Reply:
The root class of `SubforumSubscribeSection` also sets these. We can refactor these away once the other subforum page is deprecated (that's the only other place `SubforumSubscribeSection` is used)